src/QMCApp/QMCMain.cpp

```diff
@@ -331,6 +331,8 @@ void QMCMain::executeLoop(xmlNodePtr cur)
       tcur = tcur->next;
     }
   }
+  // Clean up the last driver. No further reuse needed.
```

Comment:
I think it would be better to say. Last driver is destroyed, necessary state having been transfered.  Or something like that.

Reply:
Updated.